rust/src/io/local.rs

```diff
@@ -33,7 +33,7 @@ use super::object_reader::ObjectReader;
 use crate::{Error, Result};
 
 /// Convert an [`object_store::path::Path`] to a [`std::path::Path`].
-fn to_local_path(path: &Path) -> String {
+pub fn to_local_path(path: &Path) -> String {
```

Comment:
Does this need to be public still? If we need it elsewhere, can we make it `pub(crate)` instead, so it's only public within our codebase (and not part of our API)?

Reply:
The compiler won't let me not have it public. Switched to `pub(crate)` as you suggested.